components/currencyfield.js

```diff
@@ -0,0 +1,26 @@
+const TextField = require('./textfield')
+const toCurrency = require('../lib/tocurrency')
+
+class CurrencyField extends TextField {
```

Comment:
This doesn't appear to do any specific validation of the input.

I agree that we want to be permissive in what we allow but is there anything we can do to help the user identify when they might at least have put something completely wrong in, e.g. if they type "three hundred pounds" we're not going to get anything from that?

Also, should we have some basic min/max/order-of-magnitude values that the developer can specify as configuration options? This might help prevent users accidentally putting in £300000 when they meant £3000.00.

Reply:
I've updated the schema to allow the specified prefix, spaces and commas, but error on other values. The invalid value remains in the field to help the user diagnose the problem.

This and above changes are ones I started a while ago, but parked due to other commitments. W could do with a conversation about what other configuration options we might find useful.